### ISSUE TITLE
Issue on entropy regularization loss

### ISSUE THREAD

**Issue by Switchsyj** [sshan-zhao/DG_via_ER#4]
Hi zhao,
    Is eq.{7}, entropy regularization loss computation in the paper correspond to your code c_loss + c_other_loss [https://github.com/sshan-zhao/DG_via_ER/blob/2082d3d1c3703c153b85b0f6ec67632839887b59/train.py#L187]?
    But sorry, I just find entropy computation by weighted cross entropy, is that correct?

**Comment by Switchsyj** [comment]
> Hi zhao,
> Is eq.{7}, entropy regularization loss computation in the paper correspond to your code c_loss + c_other_loss [https://github.com/sshan-zhao/DG_via_ER/blob/2082d3d1c3703c153b85b0f6ec67632839887b59/train.py#L187]?
> But sorry, I just find entropy computation by weighted cross entropy, is that correct?

**Comment by Switchsyj** [comment]
i.e, how to compute H(P || Q) as eq.{3}, to make an alignment with uniform distribution?

**Comment by birdkyle7918** [comment]
hello，switch，why are you so Niu B?
